processing/src/main/java/sun/reflect/MagicAccessorBridge.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package sun.reflect;
+
+/**
+ * Public bridge is needed, because MagicAccessorImpl is a package-private class.
+ */
+public class MagicAccessorBridge extends MagicAccessorImpl
```

Comment:
It creates `"sun/reflect/GroovyMagic"` class. Actually an interesting question, asked here: https://twitter.com/leventov/status/863155057113669632